src/app/home/home.dashboard.ts

```diff
@@ -152,52 +152,54 @@ export const homeDashboard: DashboardDefaultConfiguration = {
         children: [
           {
             type: 'metric-display-widget',
-            title: 'Total Calls',
+            title: 'p99 Latency',
+            subscript: 'ms',
```

Comment:
The metric display widget does not support units out of the box? 🤔

Reply:
trace-metric data source doesn't do that yet.